faker/providers/job/de_DE/__init__.py

```diff
@@ -0,0 +1,42 @@
+# coding=utf-8
+from __future__ import unicode_literals
+from .. import Provider as BaseProvider
+
+
+class Provider(BaseProvider):
+    jobs = [
+        'Altenpfleger',
```

Comment:
Is there a source for this list?

Reply:
Could you add the source as comment above the list?

Reply:
Done :)